Add passwordless test helpers (#90)
mikker/passwordless: README.md

  * [Token and Session Expiry](#token-and-session-expiry)
  * [Redirecting back after sign-in](#redirecting-back-after-sign-in)
  * [Claiming tokens](#claiming-tokens)

> * [Testing](#testing)

Reviewer: ```suggestion
* [Test helpers](#testing)
```

```
</details>

> ## Testing

Reviewer: ```suggestion
## Test helpers
```

> To test controller that are protected by a `before_action :require_user`, require Passwordless's test helpers in your test suite.

Reviewer: ```suggestion
To help with testing, a set of test helpers are provided.
```

> For `rspec`, add the following line to your `spec/rails_helper.rb` or

Reviewer: ```suggestion
If you are using RSpec, add the following line to your `spec/rails_helper.rb` or
```

> `spec/spec_helper` if `rails_helper` does not exist:

Reviewer: ```suggestion
`spec/spec_helper.rb` if `rails_helper` does not exist:
```

```ruby
require "passwordless/test_helpers"
```

For `test-unit`, add this line to your `test/test_helper.rb`:

```ruby
require "passwordless/test_helpers"
```

Then in your controller, request, and system tests/specs, you can utilize the following methods:

```ruby
passwordless_sign_in # takes you to the sign in path

> passwordless_sign_in(user) # signs you in as a user

Reviewer: I'm not sure I like this dual meaning API. Let's split it into `passwordless_sign_in(resource)` and `passwordless_sign_in_path`.
